src/app/Processor/ProcessorEdit/ProcessorEdit.tsx

```diff
@@ -66,6 +69,7 @@ export interface ProcessorEditProps {
 const ProcessorEdit = (props: ProcessorEditProps): JSX.Element => {
   const {
     existingProcessorName,
+    malformedTransformationTemplate,
```

Comment:
The new parameter in ProcessorEdit, it could be fine to add tests.

Reply:
done https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/98/commits/0fc4e8cbb71e3d66db33f5bd738e2981b75f6e40